colin/utils/cmd_tools.py

```diff
@@ -85,3 +93,30 @@ def is_rpm_installed():
     except FileNotFoundError:
         rpm_installed = False
     return rpm_installed
+
+
+def exit_after(s):
```

Comment:
I just realized you could have use Probe from conu. But since it's done already, let's use your implementation.

Reply:
Thanks for pointing it up -- I don't know about that.